mmv1/templates/terraform/property_documentation.html.markdown.tmpl

```diff
@@ -38,12 +40,6 @@
   {{- end }}
   {{- if or $.WriteOnlyLegacy $.WriteOnly }}
   **Note**: This property is write-only and will not be read from the API.
-  {{- end }}
-  {{- if and (not $.FlattenObject) $.NestedProperties }}
-  Structure is [documented below](#nested_{{ $.LineageAsSnakeCase}}).
-  {{- end }}
-  {{- if $.DeprecationMessage }}
 
-  ~> **Warning:** {{ $.DeprecationMessage }}
-  {{- end -}}
-{{ "" }}
\ No newline at end of file
+  * ~> **Note:** One of `{{ underscore $.ApiName }}` or `{{ underscore $.ApiName }}_wo` can only be set.
+  {{- end }}
```

Comment:
it looks like the handling of deprecation messages and nested structs accidentally got lost.